Week2/homework/map-filter.js

```diff
@@ -1,8 +1,7 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
+  return numbers.filter(number => number % 2 !== 0).map(number => number * 2);
```

Comment:
Is the name `number` in the map method the best name you could give??

>Think about what numbers you're filtering out in the first place!

Reply:
I think it's not. Should it be oddNumber?